extensions/coze/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "coze",
+  "title": "Coze",
+  "description": "Coze is an AI assistant that can help you with various tasks, including writing, translation, and coding. This extension allows you to use Coze directly in Raycast.",
+  "icon": "coze.png",
+  "author": "chyroc",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "preferences": [
+    {
+      "name": "api_base",
+      "title": "API Base",
+      "description": "The base URL of the Coze API",
+      "type": "textfield",
+      "required": true,
+      "default": "https://api.coze.com",
+      "placeholder": "https://api.coze.com"
```

Comment:
Will this ever change? Is a preference necessary?

Reply:
it can set to https://api.coze.cn

Reply:
or inner api host

Reply:
Makes sense